lib/Compiler.cpp

```diff
@@ -805,6 +805,7 @@ int Compile(const int argc, const char *const argv[]) {
   llvm::PassRegistry &Registry = *llvm::PassRegistry::getPassRegistry();
   llvm::initializeCore(Registry);
   llvm::initializeScalarOpts(Registry);
+  llvm::initializeClspvPasses(Registry);
```

Comment:
This likely needs done in `CompileFromSourceString` too.

Reply:
Sure.  I don't see this function tested anywhere, though.  Are there unit tests I can add to?

Reply:
Not without doing some legwork. That function is for the in process API.